chromium_src/chrome/renderer/DEPS

```diff
@@ -1,8 +1,4 @@
 include_rules = [
-  "+brave/components/ai_chat/core/common",
```

Comment:
strange, this should have failed checkdeps because they are still used in url_loader_throttle_provider_impl.cc